apps/condo/domains/common/components/containers/amplitude/AmplitudeProvider.tsx

```diff
@@ -17,9 +17,10 @@ export type BaseEventProperties = {
 }
 
 export enum AmplitudeEventType {
-    pageLoad = 'PageLoad',
-    clickLink = 'ClickLink',
-    clickButton = 'ClickButton',
+    VisitBillingAppListPage = 'VisitBillingAppListPage',
+    VisitBillingPage = 'VisitBillingPage',
+    VisitBillingAboutPage = 'VisitBillingAboutPage',
+    OpenDescriptionLink = 'OpenDescriptionLink',
```

Comment:
Missing context, where description link is opened

Reply:
It looks, like this `enum` will be overflowed by tons of values, specific to all our domains, which will blur domain-agnostic trait of `common` domain.